proxy/core/base/tcp_server.py

```diff
@@ -52,19 +52,23 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.must_flush_before_shutdown = False
         if self.flags.unix_socket_path:     # pragma: no cover
             logger.debug(
-                'Connection accepted from {0}'.format(self.work.address),
+                'Work#{0} accepted from {1}'.format(
+                    self.work.connection.fileno(), self.work.address,
+                ),
             )
```

Comment:
There are a few problems here:
1) You could use an f-string
2) It's best to use named placeholders, using numbers hurts readability because it increases the cognitive complexity (the reader would need to match numbers with the format args and count their positions each time)
3) It's an antipattern to pre-render log entries. For any use of `logging`, you should pass a string with placeholders and then, separate values. There are a few reasons for this: (1) the logging framework may be deciding not to output this log entry depending on the configuration on a completely different level so rendering it is wasteful; (2) some tools that collect stats on what logging invocations are hit use the log line to group the data and it works best with a template line being the same and not different every time the control flow hits this place.
```suggestion
            logger.debug(
                'Work#%d accepted from %s',
                self.work.connection.fileno(),
                self.work.address,
            )
```

Reply:
Applied manually, GH says file was deleted.  I moved them around